crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -354,6 +358,18 @@ where
         self.alarm_converter = AlarmConverter::Synced;
         sync_messages
     }
+
+    fn process_operation_update_messages(&mut self, message: &Message) {
+        let payload: DiscoverOp = serde_json::from_str(&message.payload_str().unwrap()).unwrap();
+        match payload.event_type {
+            EventType::ADD => {
+                self.operations.add_operation(payload.operation_name);
+            }
+            EventType::REMOVE => {
+                self.operations.remove(&payload.operation_name);
```

Comment:
Why the add and remove methods don't have similar signature? i.e both with same kind of arguments and both with the operation suffix or none. This might be seen as nitpicking, but uniformity is a key point to write code that is clear, easy to read and maintain.